messaging/rabbitmq_user.py

```diff
@@ -175,14 +198,21 @@ def set_tags(self):
         self._exec(['set_user_tags', self.username] + self.tags)
 
     def set_permissions(self):
-        cmd = ['set_permissions']
-        cmd.append('-p')
-        cmd.append(self.permissions['vhost'])
-        cmd.append(self.username)
-        cmd.append(self.permissions['configure_priv'])
-        cmd.append(self.permissions['write_priv'])
-        cmd.append(self.permissions['read_priv'])
-        self._exec(cmd)
+        for permission in self._permissions:
```

Comment:
When adding a new user, self._permissions is None.
Will meet NoneType is not iterable error.
I suggest initialize self._permissions = [] instead of None